assets/js/components/SapSystemDetails/InstanceStatus.jsx

```diff
@@ -1,31 +1,33 @@
 import React from 'react';
 import Pill from '@components/Pill/Pill';
+import { EOS_LENS_FILLED } from 'eos-icons-react';
```

Comment:
I would put this `import` before importing `Pill`.
We put the 3rd party imports at the top usually, before our own component imports